src/styles/components/typeahead.less

```diff
@@ -0,0 +1,27 @@
+.typeahead.inverse {
```

Comment:
I don't think we should couple the inverse styling to this component so tightly. Inverse should be a modifier to the base styling.